group_milestones.go

```diff
@@ -56,9 +57,10 @@ func (m GroupMilestone) String() string {
 // https://docs.gitlab.com/ce/api/group_milestones.html#list-group-milestones
 type ListGroupMilestonesOptions struct {
 	ListOptions
-	IIDs   []int  `url:"iids,omitempty" json:"iids,omitempty"`
-	State  string `url:"state,omitempty" json:"state,omitempty"`
-	Search string `url:"search,omitempty" json:"search,omitempty"`
+	IIDs                    []int  `url:"iids,omitempty" json:"iids,omitempty"`
+	State                   string `url:"state,omitempty" json:"state,omitempty"`
+	Search                  string `url:"search,omitempty" json:"search,omitempty"`
+	IncludeParentMilestones bool   `url:"include_parent_milestones,omitempty" json:"include_parent_milestones,omitempty"`
```

Comment:
I know you did not add all of these, but the `string` and `bool` fields in this struct need to be pointers. And maybe we should add `title` as well to make the list complete?

Reply:
What's the guidance for when a pointer should or should not be used?

Reply:
Add title, fixed pointers.